### DISCUSSION TITLE
Stackoverflow Icon

### DISCUSSION THREAD

**Discussion by OCram85** [jpanther/congo#80] [Ideas]
I think the stackoverflow icon /link would be nice in the author link list.

**Comment by rshmhrj** [comment]
@jpanther I can add this one if you want: https://fontawesome.com/v5.15/icons/stack-overflow?style=brands

**Reply by jpanther** [reply]
Thanks @rshmhrj, that'd be great.

**Reply by rshmhrj** [reply]
@jpanther Added https://github.com/jpanther/congo/pull/82 for your review!

**Reply by jpanther** [reply]
Merged. The new icon will be in the next release.

**Reply by OCram85** [reply]
Awesome! Thanks for the super fast implementation ;)
